config/installbundle/release-manifests/autopilot/kustomization.yaml

```diff
@@ -16,7 +16,7 @@ namespace: configconnector-operator-system
 commonLabels:
   cnrm.cloud.google.com/operator-system: "true"
 commonAnnotations:
-  cnrm.cloud.google.com/operator-version: "1.124.0-rc.1"
```

Comment:
The rest looks good. Are we sure this is correct?  "0.0.0-dev" is fine in a dev build. However I don't want that to end up in as KCC released build.

Reply:
Oh yes that's valid. I changed it thinking that it would be better for developers when they try to use their own locally built image. But it should not go into release for sure.